cli/functionary/environment.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+import click
+import requests
+
+from .config import get_config_value, save_config_value
+
+
+def get_environment_list():
+    """
+    Helper function to get the environment list from host
+
+    Args:
+        None
+
+    Returns:
+        None
+
+    Raises:
+        ClickException if bad get request
+    """
+    token = get_config_value("token")
+    teams_url = get_config_value("host") + "/api/v1/teams"
+    header = {"Authorization": f"Token {token}"}
+    response = requests.get(teams_url, headers=header)
```

Comment:
This can throw an exception.  For instance, if the host is unavailable.  You'll want to wrap it in a try / except block so that it can raise the more friendly ClickException rather than throwing a stack trace.